app/src/main/java/mozilla/lockbox/presenter/BackablePresenter.kt

```diff
@@ -1,22 +1,17 @@
 package mozilla.lockbox.presenter
 
-import io.reactivex.Observable
-import io.reactivex.rxkotlin.addTo
-import mozilla.lockbox.action.RouteAction
 import mozilla.lockbox.flux.Dispatcher
 import mozilla.lockbox.flux.Presenter
 
-interface BackableViewProtocol {
-    val backButtonTaps: Observable<Unit>
+interface BackableView {
+    // No methods yet.
 }
 
 class BackablePresenter(
-    val view: BackableViewProtocol,
+    val view: BackableView,
     val dispatcher: Dispatcher = Dispatcher.shared
 ) : Presenter() {
     override fun onViewReady() {
-        view.backButtonTaps
-                .subscribe { dispatcher.dispatch(RouteAction.Back) }
-                .addTo(compositeDisposable)
+        // NOOP
```

Comment:
do we need to override this function?

Reply:
I deleted the method, but I'm leaving this mostly for a tech debt exercise namely: "What is the point of `BackablePresenter`?".